shprofile.sh

```diff
@@ -330,6 +333,19 @@ function shpCreateProfile {
     
     mkdir -p "$profileHome"
     chmod +x "$profileHome"
+    cat > "$profileHome"/001-rcm.sh << EOF
+#!/usr/bin/env bash
+# 
+# Load all rcm-managed dotfiles for this shprofile, represented as a tag in rcm
+rcup -t $profileName
```

Comment:
This line assumes that the user already has `rcm` installed, where `shprofile` should be agnostic to any command not installed by default in a classic bash environment.